Include/internal/pycore_interp.h

```diff
@@ -191,6 +191,9 @@ struct _is {
 
     /* the initial PyInterpreterState.threads.head */
     PyThreadState _initial_thread;
+
+    // lazy imports status in runtime
+    int lazy_imports_enabled;
```

Comment:
I don't see this being initialized (to `0`) anywhere - are we relying on default initialization, and is it safe?

Reply:
After my investigation, I think it's good to initialize it in `pycore_create_interpreter` function.